BangazonAPI/Controllers/EmployeesController.cs

```diff
@@ -0,0 +1,180 @@
+﻿/*
```

Comment:
Why is the EmployeeController in the "models" PR?

Reply:
I must have made a branch off of a branch. I'll fix that.